### ISSUE TITLE
typo in doc

### ISSUE THREAD

**Issue by courtiol** [thibautjombart/adegenet#255]
In `?find.cluster`:

There is a typo in the following sentence
```
 First important result is that BIC seems for efficient than AIC and WSS to...
```
which should read
```
 First important result is that BIC seems MORE efficient than AIC and WSS to...
```
++

**Comment by zkamvar** [comment]
Thanks! If you are comfortable with github, can you open a PR for the change?

**Comment by courtiol** [comment]
will do when I find 5 min
++

**Comment by courtiol** [comment]
It did not took me a year to find 5 min, but having the new course to prepare :-)
Should be solved in PR (#273). Will close this issue if PR is integrated.
